src/server/services/wms/qgswmsrenderer.cpp

```diff
@@ -615,7 +615,7 @@ namespace QgsWms
             {
               QList<QgsMapLayer *> layersFromGroup;
 
-              for ( QgsMapLayer *layer : mContext.layersFromGroup( layer.mNickname ) )
+              for ( QgsMapLayer *layer : qgis::as_const( mContext.layersFromGroup( layer.mNickname ) ) )
```

Comment:
Same here, you need to take a const copy. as_const is not meant to be on rvalues (https://stackoverflow.com/questions/39051460/why-does-as-const-forbid-rvalue-arguments)

Reply:
thanks @m-kuhn